ports/outcome/portfile.cmake

```
@@ -1,13 +1,55 @@
-# header-only library
-vcpkg_from_github(
-    OUT_SOURCE_PATH SOURCE_PATH
-    REPO ned14/outcome
-    REF 34f3bd55e2bcaf246cb79efe64a5600e89b91b66 #v2.1.3
-    SHA512 83eba50e2095e7c768dacb3af5f82db117c3451f1d5bc2f73d716608d56f7b73006ec33d0f3842fdefd076f0e82b72ece5777868712f75e83eac93aa8adf351c
-    HEAD_REF develop
+set(OUTCOME_VERSION v2.2.0)
+set(OUTCOME_GITCOMMIT ac552d1c69ef556a1327393a0c56092517ff92db)
+set(OUTCOME_HASH 2d27434bcb2d5bc27015fedc221eafd221acb7c873277c4f0c78bc6aa657e6e962f0458e64b2de5f695600a5b15559cd0d3602f276f35c414dfafb34c2214e8c)
+
+# Use Outcome's all sources tarball, not its github repo. The tarball includes all dependencies.
+vcpkg_download_distfile(
+  ALL_SOURCES_TARBALL
+  URLS "https://github.com/ned14/outcome/releases/download/all_tests_passed_${OUTCOME_GITCOMMIT}/outcome-v2-all-sources-${OUTCOME_GITCOMMIT}.tar.xz"
+  FILENAME "outcome-v2-all-sources-${OUTCOME_GITCOMMIT}.tar.xz"
+  SHA512 ${OUTCOME_HASH}
+)
+vcpkg_extract_source_archive_ex(
+  SKIP_PATCH_CHECK
+  OUT_SOURCE_PATH SOURCE_PATH
+  ARCHIVE "${ALL_SOURCES_TARBALL}"
+  REF "outcome-${OUTCOME_GITCOMMIT}"
+)
+
+# Use Outcome's own build process, skipping examples and tests, bundling the embedded quickcpplib
+# instead of git cloning from latest quickcpplib.
+vcpkg_configure_cmake(
+    SOURCE_PATH "${SOURCE_PATH}"
+    PREFER_NINJA
+    OPTIONS
+        -DPROJECT_IS_DEPENDENCY=On
+        -DOUTCOME_BUNDLE_EMBEDDED_QUICKCPPLIB=On
 )
 
-file(GLOB_RECURSE OUTCOME_HEADERS "${SOURCE_PATH}/single-header/*.hpp")
-file(INSTALL ${OUTCOME_HEADERS} DESTINATION ${CURRENT_PACKAGES_DIR}/include)
+vcpkg_install_cmake()
+
+# Looks like vcpkg_fixup_cmake_targets() can't be run twice, so do this by hand
+file(RENAME "${CURRENT_PACKAGES_DIR}/lib/cmake/quickcpplib" "${CURRENT_PACKAGES_DIR}/share/quickcpplib")
+file(RENAME "${CURRENT_PACKAGES_DIR}/share/cmakelib" "${CURRENT_PACKAGES_DIR}/share/quickcpplib/cmakelib")
+file(RENAME "${CURRENT_PACKAGES_DIR}/share/scripts" "${CURRENT_PACKAGES_DIR}/share/quickcpplib/scripts")
+
+# Must come AFTER the above, as it appears to hose the lib/cmake directory
+vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/${PORT}" TARGET_PATH "share/${PORT}")
```

```suggestion
file(RENAME "${CURRENT_PACKAGES_DIR}/share/cmakelib" "${CURRENT_PACKAGES_DIR}/share/quickcpplib/cmakelib")
file(RENAME "${CURRENT_PACKAGES_DIR}/share/scripts" "${CURRENT_PACKAGES_DIR}/share/quickcpplib/scripts")

vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/quickcpplib TARGET_PATH share/quickcpplib)
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/outcome)
```
In fact it can be, but only for `config.cmake` or `targets.cmake`.

---

I tried using `vcpkg_fixup_cmake_targets()` twice exactly as you have it, as that seems more logical. But as I mentioned in the comment, that function hoses the lib/cmake directory, which causes the second call to fail.